extensions/wikia/AdEngine/js/slot/adSlot.js

```diff
@@ -55,9 +55,9 @@ define('ext.wikia.adEngine.slot.adSlot', [
 		var cssSelector = '#' + slotName + ' > .provider-container:not(.hidden) div[id*="_container_"] > iframe',
 			iframe = doc.querySelector(cssSelector);
 
-		if (!iframe && adBlockDetection && adBlockDetection.isBlocking()) {
-			iframe = getRecoveredIframe(slotName);
-		}
+		// if (!iframe && adBlockDetection && adBlockDetection.isBlocking()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)